docs/client/wn-oasis.md

```diff
@@ -0,0 +1,126 @@
+Configuring a Site to Use the Worker Node Client Software From OASIS
+====================================================================
+
+
+About This Guide
+----------------
+
+The *OSG Worker Node Client* is a collection of software components that is expected to be added to every worker node that can run OSG jobs. It provides a common environment and a minimal set of common tools that all OSG jobs can expect to use. See the [reference section](#Reference) below for contents of the Worker Node Client.
+
+It is possible to install the Worker Node Client software in a variety of ways, depending on what works best for distributing and managing software at your site:
+
+-   Use the Worker Node Client software from OASIS (this guide) - useful when [OASIS](cvmfs.md) is already mounted on your worker nodes
```

Comment:
I don't believe the CVMFS doc has been fully converted yet, we should refer to the TWiki doc until then